android-stellar-sdk/build.gradle

```diff
@@ -22,10 +22,8 @@ dependencies {
     implementation 'com.moandjiezana.toml:toml4j:0.5.1'
     implementation 'com.google.guava:guava:18.0'
     implementation 'com.google.code.gson:gson:2.4'
-    implementation('com.github.heremaps:oksse:master-SNAPSHOT') {
```

Comment:
why not using external lib and instead copy paste code ?

Reply:
because it's a small code, the lib is not much stable (no version for instance), and I made some changes to make it work and to adjust to our case

Reply:
I prefer to inline it instead of forking etc. though it might worth opening a PR for them

Reply:
changed to our forked version at https://github.com/kinfoundation/oksse